sasdata/dataloader/loader.py

```diff
@@ -363,14 +364,26 @@ def associate_file_reader(self, ext: str, loader) -> bool:
         """
         return self.__registry.associate_file_reader(ext, loader)
 
-    def load(self, file: str, format: Optional[str] = None) -> Union[list, Exception]:
+    def load(self, file_path_list: Union[List[str], str],
+             format: Optional[Union[List[str], str]] = None
+             ) -> Union[List[Union[Data1D, Data2D]], Exception]:
```

Comment:
You are always returning a list. Exceptions are being eaten, which is bad. If the registry load is set up to return rather than raise exceptions, then the return signature would be `List[Union[Data1D, Data2D, Exception]]`.